mmpose/models/detectors/top_down.py

```diff
@@ -228,22 +224,30 @@ def process_head(self, output, img, img_metas, return_heatmap=False):
                 output_flipped = self.keypoint_head(output_flipped)
             if isinstance(output_flipped, list):
                 output_flipped = output_flipped[-1]
-            output_flipped = flip_back(
-                output_flipped.detach().cpu().numpy(),
-                flip_pairs,
-                target_type=self.target_type)
+            output_flipped = flip_back(output_flipped.detach().cpu().numpy(),
+                                       flip_pairs)
```

Comment:
Why `target_type` is deleted?

Reply:
I guess there's something wrong when I was rebasing.

Reply:
I didn't do that. Seems that there's something wrong when I was rebasing.